src/Integration.UnitTests/Binding/CSharpVBBindingConfigProviderTests.cs

```diff
@@ -144,32 +132,29 @@ public async Task GetConfig_NoSupportedActiveRules_ReturnsNull()
                 string.Format(Strings.NoSonarAnalyzerActiveRulesForQualityProfile, validQualityProfile.Name, Language.VBNET.Name));
             builder.Logger.AssertOutputStrings(expectedOutput);
 
-            builder.AssertRuleSetGeneratorNotCalled();
+            builder.AssertGlobalConfigGeneratorNotCalled();
         }
 
         [TestMethod]
-        public async Task GetConfig_ReturnsCorrectRuleset()
+        public async Task GetConfig_ReturnsCorrectGlobalConfig()
         {
             var builder = new TestEnvironmentBuilder(validQualityProfile, Language.VBNET)
             {
                 ActiveRulesResponse = validRules,
                 InactiveRulesResponse = emptyRules,
                 PropertiesResponse = anyProperties,
-                RuleSetGeneratorResponse = validRuleSet,
+                GlobalConfigGeneratorResponse = globalConfig,
```

Comment:
I'd set the `GlobalConfigGeneratorResponse` to a hard-coded string, and check the expected content is returned after line 174.